package.json

```diff
@@ -11,14 +11,14 @@
     "prepare": "husky"
   },
   "devDependencies": {
-    "@repo/eslint-config": "*",
-    "@repo/typescript-config": "*",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-prettier": "^5.1.3",
-    "husky": "^9.0.7",
-    "lint-staged": "^15.2.0",
-    "prettier": "^3.1.0",
-    "turbo": "^2.0.11"
+    "@repo/eslint-config": "^0.0.0",
+    "@repo/typescript-config": "^0.0.0",
```

Comment:
Can you switch these values to "*", I am not sure btw, if new version of the turborepo coverted this, then leave it. Write just comment here.